README.md

```diff
@@ -6,47 +6,30 @@ direnv -- unclutter your .profile
 [![latest packaged version(s)](https://repology.org/badge/latest-versions/direnv.svg)](https://repology.org/project/direnv/versions)
 [![Support room on Matrix](https://img.shields.io/matrix/direnv:numtide.com.svg?label=%23direnv%3Anumtide.com&logo=matrix&server_fqdn=matrix.numtide.com)](https://matrix.to/#/#direnv:numtide.com)
 
-`direnv` is an extension for your shell. It augments existing shells with a
-new feature that can load and unload environment variables depending on the
-current directory.
+`direnv` is an extension for your unix shell. It augments existing shells with a
+new feature that can **load and unload environment variables** depending on the
+directory you are in.
 
 ## Use cases
 
-* Load [12factor apps](https://12factor.net/) environment variables
+Primary use-cases **hacking on nix based software**:
```

Comment:
I don't know that this is the primary use case.

The principle of direnv is that it's an agnostic interface.
Many people use direnv with asdf, homebrew, or just standalone.

Reply:
I trust you here, as I said, I don't have a clue about the direnv users' environments.